### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by tommartensen**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -107,8 +107,180 @@
...
+
+.alignItemsFlexEnd {
+	align-items: flex-end
+}
+
+.textAlignCenter {
+	text-align: center
+}
+
+.rotate90 {
```
Comment (frederike-ramin) [inline src/App.css:174]: This class is doubled. (see above)

File: src/DetailView/DetailView.js [hunk 2]
```diff
@@ -14,9 +13,9 @@ import FilterBar from "./../Utils/FilterBar";
 import Utils from "./../Utils/Utils";
 import {Toggle} from "material-ui";
 import LoadingAnimation from "../Utils/LoadingAnimation";
-import { css } from 'aphrodite';
 import "./DetailView.css";
 import "./../App.css";
+import AppStyles from "../AppStyles";
```
Comment (frederike-ramin) [inline src/DetailView/DetailView.js:18]: Why do you need AppStyles here but not aphrodite?

File: src/Header.js [hunk 3]
```diff
@@ -66,14 +66,14 @@ class Header extends Component {
...
-		let appBarStyle = css(AppStyles.positionFixedTop);
+		let appBarStyle = "positionFixedTop";
 		if (this.props.status) {
 			const statusColor = StyleSheet.create({
 				color: {
 					borderColor: Utils.getColorForStatus(this.props.status)
 				}
 			});
-			appBarStyle = css(AppStyles.headerBorderDetail, statusColor.color, AppStyles.positionFixedTop);
+			appBarStyle = "headerBorderDetail positionFixedTop" + css(statusColor.color);
```
Comment (frederike-ramin) [inline src/Header.js]: Color is wrog and header not sticky anymore.

**Review by frederike-ramin — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (tommartensen) [reply src/App.css:174]: thx

File: src/DetailView/DetailView.js (hunk 2 above)
Comment (tommartensen) [reply src/DetailView/DetailView.js:18]: to style the switch color

**Review by frederike-ramin — APPROVED** [review]

**Pull request merged**